### ISSUE TITLE
metadata generation on backend

### ISSUE THREAD

**Issue by sxysun** [rpalakkal/teleport-rs#13]
Currently if I list my account link NFT on OpenSea or Sudoswap, the buyer, without checking the chain, will have to take my word for it that the NFT allow them to tweet from `my account` with `safeguard text`. It would be ideal to provide a way for account link NFT buyers to easily check those properties without having to read events from the chain.

We can either do this by having the backend credibly generate metadata on each mint, or just have an external helper tool website that reads events.

**Comment by amiller** [comment]
Example format
```json
{
  "description": "Allow you to post once from @sxysun1 account if your post pass the following check by the LLM: 'no restriction, allow anything'", 
  "image": "https://storage.googleapis.com/opensea-prod.appspot.com/puffs/3.png", 
  "name": "Teleport to @sxysun1",
  "attributes": [
    {
      "trait_type": "x_pfp", 
      "value": "https://pbs.twimg.com/profile_images/1568852016394371072/4DO1pfje_400x400.jpg"
    }, 
    {
      "trait_type": "x_username", 
      "value": "sxysun"
    }, 
    {
      "trait_type": "x_handle", 
      "value": "@sxysun"
    }, 
    {
      "trait_type": "created on", 
      "value": "09/08/2024"
    }, 
    {
      "trait_type": "LLM safeguard", 
      "value": "no restriction, allow anything"
    }
  ]
}
```

**Comment by amiller** [comment]
Example of how `tokenURI(uint tokenid)` can be used to display metadata directly from the contract
https://chatgpt.com/c/670435d6-cd14-8009-9f8d-a2a413acec01

**Comment by amiller** [comment]
fixed with #27
